pkg/scheduler/scheduler_test.go

```diff
@@ -200,10 +198,8 @@ func TestSchedulerCreation(t *testing.T) {
 		eventBroadcaster.NewRecorder(scheme.Scheme, "scheduler"),
 		kubeschedulerconfig.SchedulerAlgorithmSource{Provider: &testSource},
 		stopCh,
-		emptyPluginRegistry,
-		nil,
-		emptyPluginConfig,
-		WithBindTimeoutSeconds(defaultBindTimeout))
+		WithBindTimeoutSeconds(defaultBindTimeout),
```

Comment:
Is there a specific reason for not using the default `BindTimeoutSeconds` defined in the scheduler?
https://github.com/kubernetes/kubernetes/blob/0960c74c3788b1724bd7e7b9933bc49c7e5b5afa/pkg/scheduler/scheduler.go#L50-L55

Reply:
I don't think so, I removed the option.